readme.md

```diff
@@ -91,18 +98,14 @@ Xin chào thế giới. This is Hello World in Vietnamese.
 \end{document}
 ```
 
-Here we use two packages `usepackage[T5]{fontenc}` and `usepackage[utf8]{inputenc}` . This is really simple to understand as the package will import font encoders to display your content correctly. If you are using TexMaker this is what the above code display :
-
-![](http://i.imgur.com/OUSPekM.png)
-
-vs without the packages :package::  
-
-![](http://i.imgur.com/Upd47xH.png)  
+The default font (Latin Modern) does not support all characters. You can,

```

Comment:
The default font is Computer Modern, not Latin Modern. Latin Modern is used when `lmodern` is loaded as a package.